yarn.lock

```diff
@@ -1586,18 +1586,6 @@
   resolved "https://registry.yarnpkg.com/@napi-rs/cli/-/cli-2.16.1.tgz#912e1169be6ff8bb5e1e22bb702adcc5e73e232b"
   integrity sha512-L0Gr5iEQIDEbvWdDr1HUaBOxBSHL1VZhWSk1oryawoT8qJIY+KGfLFelU+Qma64ivCPbxYpkfPoKYVG3rcoGIA==
 
-"@noble/curves@1.3.0", "@noble/curves@~1.3.0":
```

Comment:
(We already discussed on slack but putting here so it's clear this PR is being looked at)

This PR seems to only remove about 1/3 of the dependencies that the `web3-validator` library added. It's not clear to me if these are now being used by keccak but I would certainly hope not. Might be easier to have this PR also revert that original commit that added the dependency or something, not sure, requires a bit more investigation. If these are indeed all being used by keccak, we should probably just surface some bindings to the rust keccak library instead of using a JS one, since we already introduced that transitively for FishHash

Reply:
Per conversation in slack - these extra deps are from keccak, so we'll just merge this for now as it is an improvement, and then surface rust bindings to keccak in a following PR later